### DISCUSSION TITLE
Why Github Dark Dimmed from github-vscode-theme colours much more?

### DISCUSSION THREAD

**Discussion by mloskot** [projekt0n/github-nvim-theme#309] [Q&A] [answered]
I'm comparing how the theme works in NeoVim (left) versus the GitHub' Dark Dimmed in VS Code (right), and as a NeoVim rookie I'm not quite clear:
am I missing anything, any configuration, any add-ins or is such difference expected?

![image](https://github.com/projekt0n/github-nvim-theme/assets/80741/2dac4377-b7a7-4e07-9b44-b21c17484c02)

My [.config/nvim/init.lua](https://github.com/mloskot/dotfiles/blob/9e1e9d8cd43d1f65d439e2cfc660b9c947da3748/nvim/init.lua) is (still) very minimal, so I assume nothing there should affect how the theme looks. I've read the theme's README.md here and I do not see any missing configuration in my setup.

I'd appreciate if anyone could help me understand if and how to achieve the VS Code colouring of at least Go syntax :)

**Comment by tmillr** [comment, marked as answer]
Currently this colorscheme closely follows the real/original GitHub themes visible here on the website UI, not vscode's approximation of it. Subjectively and objectively, I believe this is the right way to go.

Here's an example of what it looks like for Go:

```go
package main

import "fmt"

func hello(name string) {
    message := fmt.Sprintf("Hi %v", name)
}
```

The colorscheme is highly-configurable, so if you really want it to look more like vscode (as opposed to the actual GitHub theme here on the website), you are free to configure it that way locally (and it should be fairly straightforward, just make sure to read the docs). I can provide pointers and examples if you need help with configuration, just let me know.

With that said, the highlighting has improved and become more accurate over time, and it should look better out-of-the-box than what you've shown here. Here's what I am getting right now:

<img width="413" alt="Screenshot 2024-07-12 at 4 58 44 AM" src="https://github.com/user-attachments/assets/32968086-d079-4d3a-ac1f-af4f62d84570">

You can see that it's very close to the actual theme (visible in the markdown code block above).
